test/geometry/LinearPropagator.test.cc

```diff
@@ -92,11 +92,11 @@ TEST_F(LinearPropagatorHostTest, track_line)
 
         auto step = propagate();
         EXPECT_FALSE(geo.is_outside());
-        EXPECT_EQ(VolumeId{10}, geo.volume_id()); // World
+        EXPECT_EQ(VolumeId{3}, geo.volume_id()); // World
 
         step = propagate();
         EXPECT_SOFT_EQ(7., step.distance);
-        EXPECT_EQ(VolumeId{3}, step.volume); // World -> Envelope
+        EXPECT_EQ(VolumeId{2}, step.volume); // World -> Envelope
```

Comment:
The comments in lines `79` and `159` state that `VolumeId{2}` represents `Shape1 -> Envelope`, while the comment here says `World -> Envelope`.  Could you clarify this?

Reply:
Even better, since this is a higher-level test than the geometry itself, it would be better for readability and maintainability  to directly compare the expected volume names rather than the raw volume ids:
```suggestion
        EXPECT_EQ("Envelope", geom.id_to_label(step.volume));
```

Reply:
> Even better, since this is a higher-level test than the geometry itself, it would be better for readability and maintainability to directly compare the expected volume names rather than the raw volume ids:

Thanks, I like this a lot!  Implementing...

Reply:
> The comments in lines `79` and `159` state that `VolumeId{2}` represents `Shape1 -> Envelope`, while the comment here says `World -> Envelope`. Could you clarify this?

`VolumeId{2}` corresponds to the "Envelope" volume, which is the name of the "post-step" volume in both cases.